### ISSUE TITLE
Duplicate IDs on meta box partial page

### ISSUE THREAD

**Issue by tfrommen** [WordPress/gutenberg#4713]
## Issue Overview
The `the_gutenberg_metaboxes` function [loops over three _locations_, and generates markup with IDs](https://github.com/WordPress/gutenberg/blob/f2bbf5f48aefda63e00b40ea8362a10934dacca9/lib/meta-box-partial-page.php#L308-L329) (e.g., `#poststuff`). This means, after looping through, there are multiple elements with the same ID. This is bad, of course.

Not sure what you wanted to achieve here, but this should be fixed, in my opinion.

## Expected Behavior
IDs are unique.

## Current Behavior
IDs are not unique.

## Possible Solution
Don't render markup with IDs in a loop.

**Comment by youknowriad** [comment]
In the original version we used to generate separate HTMLs for each metabox location. and I guess the idea is to have the closest possible markup to the old classic editor metaboxes.

I'd be open to removing these if there's no styling or script issues. Might have to check with several plugins to find out.

**Comment by youknowriad** [comment]
fixed in #4697
